src/main/java/seedu/duke/Parser.java

```diff
@@ -11,7 +11,18 @@
      * @return whether the program should continue running after processing the given user input
      */
     public static boolean parseUserInput(String userInput) {
-        String[] userInputSplit = userInput.split(" ", 2);
+        //userInputSplit guranteed to be length 2. UserInputSplit[1] is null by default
+        String[] userInputSplit = new String[2];
+        String[] rawInput = userInput.split(" ", 2);
+
+        if (rawInput.length == 2) {
+            userInputSplit[0] = rawInput[0];
+            userInputSplit[1] = rawInput[1];
+        } else {
+            userInputSplit[0] = rawInput[0];
+        }
+
+
         String inputCommand = userInputSplit[0].toLowerCase();
```

Comment:
userInputSplit[0] will always equal to rawInput[0] right? maybe can put on top and just make String inputCommand = rawInput[0].toLowerCase() and remove the else portion? can cut a few lines of codes i think